ghost/core/core/server/api/endpoints/utils/validators/input/settings.js

```diff
@@ -64,14 +65,26 @@ module.exports = {
             }
 
             if (setting.key === 'announcement_visibility') {
-                const visibilityValue = setting.value;
+                // NOTE: safe to parse because of array validation up top
+                const visibilityValues = JSON.parse(setting.value);
 
-                if (!['public', 'visitors', 'members', 'paid'].includes(visibilityValue)) {
-                    const visibilityError = new ValidationError({
-                        message: tpl(messages.invalidAnnouncementVisibilityValueReceived),
-                        property: setting.key
+                // NOTE: combination of 'free_members' & 'paid_members' makes just a 'members' filter
```

Comment:
If paid membership is not enabled, can we send just `free_members` filter, or should it be  `free_members` & `paid_members`?

Reply:
I think we can make a synthetic variable for the API Client saying just `'members'`, which the backend will transform under the hood to handle the logic. Would be much nicer for the API client to understand rather than digging into the details of "what to combine in which order". What do you think?

Reply:
It sounds good, will be easier to handle it on the client with such approach 🙌